### PR TITLE
Workaround #2489: Don't run Converse tests when CUDA is enabled

### PR DESCRIPTION
**Pull request by evan-charmworks**
Needs to be tested

### CODE REVIEW

**Comment by evan-charmworks** [comment]
Testing now

**Comment by evan-charmworks** [comment]
Tested and confirmed on talent, ready for review

File: benchmarks/Makefile [hunk 1]
```diff
@@ -1,11 +1,16 @@
+-include ../include/conv-mach-opt.mak
+
 # Temporary hack that needs to be removed / replaced
```
Comment (matthiasdiener) [inline benchmarks/Makefile]: What is the 'temporary hack' referring to now? (also in the other files)

**Comment by evan-charmworks** [comment]
`OPTS += $(OPTSATBUILDTIME)`

File: benchmarks/Makefile (hunk 1 above)
Comment (evan-charmworks) [reply benchmarks/Makefile]: `OPTS += $(OPTSATBUILDTIME)`

**Pull request merged**